pkg/kubernetes/types.go

```diff
@@ -36,7 +36,7 @@ const (
 
 const (
 	// DefaultKubeEventResyncInterval is the default resync interval for k8s events
-	DefaultKubeEventResyncInterval = 30 * time.Second
+	DefaultKubeEventResyncInterval = 0 * time.Second
```

Comment:
I think its good to have the k8s resync but at a much lower frequency (1 minute or so). Without the resync, if for some reason an event is not propagated to proxies (ex. due to network errors or stream disconnects), the resyncs will help trigger notifications for the latest state of the object. 

I do think that the resyncs should be queued and not simply broadcast all the time because the resyncs for the different informers will be interleaved in time and trigger too much churn.

Reply:
discussed with @eduser25 offline. We are on the same page.